packages/tasks/src/tasks/translation/data.ts

```diff
@@ -57,12 +58,12 @@ const taskData: TaskDataCustom = {
 			id: "Iker/Translate-100-languages",
 		},
 		{
-			description: "An application that can translate between English, Spanish and Hindi.",
-			id: "EuroPython2022/Translate-with-Bloom",
+			description: "An application that can translate between many languages.",
+			id: "Geonmo/nllb-translation-demo",
 		},
 	],
 	summary: "Translation is the task of converting text from one language to another.",
-	widgetModels: ["t5-small"],
+	widgetModels: ["facebook/nllb-200-1.3B"],
```

Comment:
Does the API work for this model? I see it's disabled

Reply:
Changed this